postgres/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM postgres:15-bookworm AS builder
```

Comment:
Why do we need our dockerfile for postgres ?

Reply:
We need it to build the ivm extension